FAQ.md

```diff
@@ -85,11 +85,14 @@ Here are some suggestions for when you could use `dep` or `go get`:
 -[@sdboyer in #376](https://github.com/golang/dep/issues/376#issuecomment-294045873)
 
 ## Why is `dep` ignoring a version constraint in the manifest?
-Only your project's directly imported dependencies are affected by a `dependencies` entry
+Only your project's directly imported dependencies are affected by a `constraint` entry
 in the manifest. Transitive dependencies are unaffected.
 
 Use an `overrides` entry for transitive dependencies.
 
+To pin a version of direct dependency in manifest, prefix the version with `=`.
+e.g. `version = "=x.y.z"`
```

Comment:
I just realized that we don't explain how we interpret a version without an operator... 😊 Do you mind adding this blurb as well?

```
By default, when you specify a version without an operator, such as `~` or `=`, `dep` automatically adds a caret operator, `^`. The caret operator pins the left-most non-zero digit in the version. For example:

^1.2.3 means 1.2.3 <= X < 2.0.0
^0.2.3 means 0.2.3 <= X < 0.3.0
^0.0.3 means 0.0.3 <= X < 0.0.4
```